docs/articles/baseplatform/variant-concept-in-yocto.rst

```diff
@@ -36,6 +36,15 @@ your layer. The variant directories should go into
   as ``bblayers.conf.sample``, but should contain the ``local.conf`` needed for
   the current variant.
 
+If many variants are using the same directives, it is recommended to have a
+``common`` directory with these directives in files that can then be included by
```

Comment:
This is a bit hard to read. Maybe "...a ``common`` directory containing configuration files with these shared directives. These files can then...".
It's still a bit convoluted, but I cannot come up with anything better.

Reply:
Updated.